data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg

```diff
@@ -125,7 +125,9 @@
         side=1
         controller=human
         recruit=Thug,Poacher,Spearman,Bowman,Footpad
-        gold=0 #player has large carryover here, is handled in prestart event
+        # player is supposed to have a large carryover here (handled in prestart event), but just in case:
+        {GOLD 20 10 0}
+        {INCOME 2 1 0}
```

Comment:
this makes the game harder on NORMAL and much harder on HARD by lowering the income

Reply:
`{4 3 2}` then?

Reply:
> `{4 3 2}` then?

I went with this in [0149764](https://github.com/wesnoth/wesnoth/pull/6057/commits/014976436016a4f09260a5e12fecc066999ed833).